test/conformance/sysctl_test.go

```diff
@@ -0,0 +1,61 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"testing"
+
+	"github.com/knative/pkg/test/logging"
+	"github.com/knative/serving/test"
+)
+
+// TestShouldHaveSysctlReadOnly verifies that the /proc/sys filesystem mounted within the container
+// is read-only.
+func TestShouldHaveSysctlReadOnly(t *testing.T) {
+	logger := logging.GetContextLogger(t.Name())
+	clients := setup(t)
+	ri, err := fetchRuntimeInfo(clients, logger, &test.Options{})
+	if err != nil {
+		t.Fatalf("Error fetching runtime info: %v", err)
+	}
+
+	mounts := ri.Host.Mounts
+
+	for _, mount := range mounts {
+		if mount.Error != "" {
```

Comment:
It seems we always return `Name`, so probably log it as well?

/me déjà vue :open_mouth:

Reply:
We don't have a name on mounts. We do have a path which is not relevant in the error case.

I will add for the other test failures.